tpchgen/src/dates.rs

```diff
@@ -222,4 +255,23 @@ mod test {
             assert_eq!(format_ymd(y, m, dy), date.to_string());
         }
     }
+
```

Comment:
I did a quick double check and it looks good to me 👍 

![Screenshot 2025-03-31 at 6 34 07 AM](https://github.com/user-attachments/assets/5d505f38-d25f-4351-95fd-7de63b9fe417)